etc/profile-a-l/gtk-youtube-viewers-common.profile

```diff
@@ -0,0 +1,23 @@
+# Firejail profile for gtk-youtube-viewer clones
+# Description: common profile for Trizen's gtk Youtube viewers
+# This file is overwritten after every install/update
+# Persistent local customizations
+include gtk-youtube-viewers-common.local
+# Persistent global definitions
+# added by caller profile
+#include globals.local
+
+ignore quiet
+
```

Comment:
This is confusing since it attemps to undo something in a very specific include
that is intended to be included after this one.  Also, `quiet` is supposed to
be the first command in a profile, so it would consistent to do the same for
`ignore quiet`.

Move it back to the original profiles, with a blank line before `private-bin`.
Example:

```
ignore quiet

private-bin gtk-lbry-viewer
```